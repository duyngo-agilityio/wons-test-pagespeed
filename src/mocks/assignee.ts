import { TRole } from '@/types';

export const MOCK_ASSIGNEES = {
  data: [
    {
      id: 1,
      attributes: {
        name: 'John Doe',
        avatar: '/avatar1.jpg',
        email: 'john@example.com',
        username: 'john_doe',
        fullName: 'John Doe',
        password: 'password123',
        role: 'User' as unknown as TRole,
        token: 'token123',
      },
    },
    {
      id: 2,
      attributes: {
        name: 'Jane Smith',
        avatar: '/avatar2.jpg',
        email: 'jane@example.com',
        username: 'jane_smith',
        fullName: 'Jane Smith',
        password: 'password123',
        role: 'User' as unknown as TRole,
        token: 'token123',
      },
    },
  ],
};
