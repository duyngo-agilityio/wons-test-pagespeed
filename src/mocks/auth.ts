// Types
import { ISignUpFormData } from '@/types';

export const SIGN_UP_FORM_DATA_MOCK: ISignUpFormData = {
  fullName: 'John Doe',
  username: 'johndoe',
  email: 'john.doe@example.com',
  password: 'Abcd@0987',
};

export const MOCK_SUCCESS_SIGN_UP_RES = {
  data: { user: { ...SIGN_UP_FORM_DATA_MOCK, id: '1' } },
  totalCount: 0,
};
