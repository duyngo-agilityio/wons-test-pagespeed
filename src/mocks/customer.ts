// Models
import { ICustomer } from '@/models';

// Types
import { TableColumnType } from '@/types';

export const PRODUCT_TABLE_COLUMNS_MOCK: TableColumnType<Partial<ICustomer>>[] =
  [
    {
      header: 'Name',
      accessor: 'name',
      isSort: true,
    },
    {
      header: 'FirstName',
      accessor: 'firstName',
      isSort: true,
    },
    {
      header: 'LastName',
      accessor: 'lastName',
      isSort: true,
    },
    {
      header: 'Email',
      accessor: 'email',
      isSort: true,
    },
  ];

export const PRODUCT_MOCK = [
  {
    id: '1',
    name: 'John Doe',
    firstName: 'John',
    lastName: 'Doe',
    email: 'john.doe@example.com',
    phone: '123-456-7890',
    job: 'Software Engineer',
    address: '123 Main St, City, Country',
    avatar: 'https://example.com/avatar1.jpg',
  },
  {
    id: '2',
    name: 'Jane Smith',
    firstName: 'Jane',
    lastName: 'Smith',
    email: 'jane.smith@example.com',
    phone: '098-765-4321',
    job: 'Project Manager',
    address: '456 Elm St, City, Country',
    avatar: 'https://example.com/avatar2.jpg',
  },
  {
    id: '3',
    name: 'Alice Johnson',
    firstName: 'Alice',
    lastName: 'Johnson',
    email: 'alice.johnson@example.com',
    phone: '555-666-7777',
    job: 'UX Designer',
    address: '789 Oak St, City, Country',
    avatar: 'https://example.com/avatar3.jpg',
  },
  {
    id: '4',
    name: 'Bob Williams',
    firstName: 'Bob',
    lastName: 'Williams',
    email: 'bob.williams@example.com',
    phone: '111-222-3333',
    job: 'Data Scientist',
    address: '101 Pine St, City, Country',
    avatar: 'https://example.com/avatar4.jpg',
  },
  {
    id: '5',
    name: 'Charlie Brown',
    firstName: 'Charlie',
    lastName: 'Brown',
    email: 'charlie.brown@example.com',
    phone: '444-555-6666',
    job: 'Marketing Specialist',
    address: '202 Birch St, City, Country',
    avatar: 'https://example.com/avatar5.jpg',
  },
];
