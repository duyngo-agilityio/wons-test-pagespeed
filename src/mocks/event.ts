export const EVENT_MOCKS = {
  title: 'Test Event',
  status: 'active',
  location: 'New York',
  date: new Date('2024-10-14'),
  startTime: '10:00',
  endTime: '12:00',
  users_permissions_users: [1, 2],
};

export const EVENTS_MOCKS = [
  {
    id: 1,
    attributes: {
      title: 'Test Event',
      status: 'active',
      location: 'New York',
      date: new Date('2024-10-14'),
      startTime: '10:00',
      endTime: '12:00',
      users_permissions_users: [1, 2],
    },
  },

  {
    id: 2,
    attributes: {
      title: 'Implement Component',
      descriptions: 'Implement for all component',
      date: '2024-10-20T01:30:00Z',
      time: '2024-10-20T01:30:00Z',
      createdAt: '2024-10-22T09:46:11.546Z',
      updatedAt: '2024-10-22T09:46:16.769Z',
      publishedAt: '2024-10-22T09:46:16.768Z',
    },
  },
];

export const EVENTS_MOCKS_WIDTH_USERS_PERMISSIONS = {
  title: 'Test Event',
  status: 'active',
  location: 'New York',
  date: new Date('2024-10-14'),
  startTime: '10:00',
  endTime: '12:00',
  users_permissions_users: {
    data: [
      {
        id: 3,
        attributes: {
          username: 'superadmin',
          email: 'admin1@gmail.com',
          provider: 'local',
          confirmed: false,
          blocked: false,
          fullName: 'Super Admin',
          createdAt: '2024-08-28T04:29:52.000Z',
          updatedAt: '2024-10-11T02:14:38.208Z',
          avatar:
            'https://watermark.lovepik.com/photo/20211209/large/lovepik-japanese-fresh-girl-park-photo-picture_501698500.jpg',
        },
      },
    ],
  },
};
