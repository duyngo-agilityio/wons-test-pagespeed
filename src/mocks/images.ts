export const MOCK_IMAGES = [
  'https://media.wired.com/photos/6500ad57fe61eb702d721b58/1:1/w_1800,h_1800,c_limit/Apple-iPhone-15-Pro-Hero-Gear.jpg',
  'https://clickbuy.com.vn/uploads/images/2019/09/thumb_iP11_3.jpg',
];
