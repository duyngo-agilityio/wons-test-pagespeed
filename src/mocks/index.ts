export * from './customer';
export * from './recentServices';
export * from './tabs';
