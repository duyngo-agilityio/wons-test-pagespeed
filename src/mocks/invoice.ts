// Types
import {
  TInvoiceDataResponse,
  TInvoiceListResponse,
  InvoiceStatus,
  TProductInvoiceResponse,
} from '@/types';

export const MOCK_INVOICES = [
  {
    id: 1,
    attributes: {
      price: 3443,
      createdAt: '2024-08-27T04:31:42.022Z',
      updatedAt: '2024-09-10T04:15:01.906Z',
      publishedAt: '2024-09-10T04:14:54.432Z',
      quantity: 3,
      product: {
        data: {
          id: 2,
          attributes: {
            price: 454354,
            createdAt: '2024-08-27T04:21:20.293Z',
            updatedAt: '2024-09-10T04:14:40.549Z',
            publishedAt: '2024-09-10T04:14:40.546Z',
            rating: 4,
            title: 'Macbook',
            imageUrl:
              'https://images.rawpixel.com/image_800/cHJpdmF0ZS9sci9pbWFnZXMvd2Vic2l0ZS8yMDIyLTExL3JtMzYyLTAxYS1tb2NrdXAuanBn.jpg',
          },
        },
      },
    },
  },
  {
    id: 2,
    attributes: {
      price: 4543,
      createdAt: '2024-08-27T04:33:06.097Z',
      updatedAt: '2024-09-10T04:15:13.503Z',
      publishedAt: '2024-09-10T04:15:07.024Z',
      quantity: 1,
      product: {
        data: {
          id: 1,
          attributes: {
            price: 3443,
            createdAt: '2024-08-27T04:20:56.562Z',
            updatedAt: '2024-09-10T04:14:24.736Z',
            publishedAt: '2024-09-10T04:14:24.732Z',
            rating: 4,
            title: 'I phone',
            imageUrl:
              'https://images.unsplash.com/photo-1505740420928-5e560c06d30e?fm=jpg&q=60&w=3000&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8Nnx8cHJvZHVjdHxlbnwwfHwwfHx8MA%3D%3D',
          },
        },
      },
    },
  },
];

export const MOCK_INVOICES_WITH_CUSTOMER: TInvoiceDataResponse[] = [
  {
    id: 4,
    attributes: {
      email: 'arroragaur@gmail.com',
      date: '2024-09-10',
      status: InvoiceStatus.Complete,
      address: '1254 Xo Viet Nghe Tinh, Da Nang',
      isSelected: true,
      invoiceId: '871345',
      imageUrl: '',
      customer: {
        data: {
          id: 3,
          attributes: {
            id: 1,
            email: 'arroragaur@gmail.com',
            phone: '(213) 555-5554',
            gender: 'male',
            address: 'danang',
            avatar:
              'https://images.pexels.com/photos/90946/pexels-photo-90946.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500',
            firstName: 'Arrora',
            job: 'UI/UX',
            lastName: ' Gaur',
            fullName: 'Arrora Gaur',
          },
        },
      },
      invoice_products: {
        data: [
          {
            id: 6,
            attributes: {
              price: 454354,
              createdAt: '2024-09-22T22:21:05.920Z',
              updatedAt: '2024-09-22T22:21:05.920Z',
              publishedAt: '2024-09-22T22:21:05.914Z',
              quantity: 1,
              product: {
                data: {
                  price: 3443,
                  createdAt: '2024-08-27T04:20:56.562Z',
                  updatedAt: '2024-09-10T04:14:24.736Z',
                  publishedAt: '2024-09-10T04:14:24.732Z',
                  rating: 4,
                  title: 'Macbook',
                  imageUrl:
                    'https://images.unsplash.com/photo-1505740420928-5e560c06d30e?fm=jpg&q=60&w=3000&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8Nnx8cHJvZHVjdHxlbnwwfHwwfHx8MA%3D%3D',
                },
              },
            },
          },
        ],
      },
    },
  },
  {
    id: 2,
    attributes: {
      email: 'annejacob@gmail.com',
      date: '2024-09-10',
      status: InvoiceStatus.Complete,
      address: 'danang',
      isSelected: false,
      invoiceId: '32434',
      imageUrl: '',
      customer: {
        data: {
          id: 6,
          attributes: {
            id: 2,
            email: 'annejacob@gmail.com',
            phone: '(325) 987-6541',
            gender: 'female',
            address: '1254 Xo Viet Nghe Tinh, Da Nang',
            avatar:
              'https://cellphones.com.vn/sforum/wp-content/uploads/2024/02/anh-avatar-cute-4.jpg',
            firstName: 'Anne ',
            job: 'UI/UX',
            lastName: 'Jacob',
            fullName: 'Anne Jacob',
          },
        },
      },
      invoice_products: {
        data: [
          {
            id: 5,
            attributes: {
              price: 454354,
              createdAt: '2024-09-22T22:21:05.920Z',
              updatedAt: '2024-09-22T22:21:05.920Z',
              publishedAt: '2024-09-22T22:21:05.914Z',
              quantity: 1,
              product: {
                data: {
                  price: 3443,
                  createdAt: '2024-08-27T04:20:56.562Z',
                  updatedAt: '2024-09-10T04:14:24.736Z',
                  publishedAt: '2024-09-10T04:14:24.732Z',
                  rating: 4,
                  title: 'I phone',
                  imageUrl:
                    'https://images.unsplash.com/photo-1505740420928-5e560c06d30e?fm=jpg&q=60&w=3000&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8Nnx8cHJvZHVjdHxlbnwwfHwwfHx8MA%3D%3D',
                },
              },
            },
          },
        ],
      },
    },
  },
];

export const MOCK_INVOICES_RESPONSE: TInvoiceListResponse = {
  data: MOCK_INVOICES_WITH_CUSTOMER,
  meta: {
    pagination: {
      page: 1,
      pageSize: 10,
      pageCount: 1,
      total: 2,
    },
  },
};

export const MOCK_INVOICE_PRODUCT_RESPONSE: TProductInvoiceResponse[] = [
  {
    id: 1,
    attributes: {
      price: 1200,
      createdAt: '2024-08-27T04:31:42.022Z',
      updatedAt: '2024-09-10T04:15:01.906Z',
      publishedAt: '2024-09-10T04:14:54.432Z',
      quantity: 1,
      product: {
        data: {
          id: 1,
          attributes: {
            id: 1,
            brand: 'Apple',
            description: 'Expensive',
            negotiable: true,
            imageUrl: 'https://example.com/image1.jpg',
            title: 'MacBook Pro',
            price: 1200,
            rating: 4.5,
          },
        },
      },
    },
  },
  {
    id: 2,
    attributes: {
      price: 1,
      createdAt: '2024-08-27T04:31:42.022Z',
      updatedAt: '2024-09-10T04:55:01.906Z',
      publishedAt: '2024-09-10T04:57:54.432Z',
      quantity: 10,
      product: {
        data: {
          id: 1,
          attributes: {
            id: 1,
            brand: 'Apple',
            description: 'Expensive',
            negotiable: true,
            imageUrl: 'https://example.com/image1.jpg',
            title: 'MacBook Pro',
            price: 1200,
            rating: 4.5,
          },
        },
      },
    },
  },
];
