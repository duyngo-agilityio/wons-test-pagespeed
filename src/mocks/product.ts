// types
import { StrapiModel } from '@/types/strapi';

// models
import { IProduct } from '@/models';

export const mockProducts: StrapiModel<IProduct>[] = [
  {
    id: 1,
    attributes: {
      title: 'Product 1',
      price: 100,
      rating: 5,
      imageUrl:
        'https://img-cdn.pixlr.com/image-generator/history/65bb506dcb310754719cf81f/ede935de-1138-4f66-8ed7-44bd16efc709/medium.webp',
    },
  },
  {
    id: 2,
    attributes: {
      title: 'Product 2',
      price: 200,
      rating: 4.5,
      imageUrl:
        'https://img-cdn.pixlr.com/image-generator/history/65bb506dcb310754719cf81f/ede935de-1138-4f66-8ed7-44bd16efc709/medium.webp',
    },
  },
  {
    id: 3,
    attributes: {
      title: 'Product 3',
      price: 200,
      rating: 4.5,
      imageUrl:
        'https://img-cdn.pixlr.com/image-generator/history/65bb506dcb310754719cf81f/ede935de-1138-4f66-8ed7-44bd16efc709/medium.webp',
    },
  },
];

export const MOCK_PRODUCTS_WITHOUT_ATTRIBUTES: Omit<IProduct, 'id'>[] = [
  {
    title: 'Product 1',
    price: 100,
    rating: 5,
    imageUrl:
      'https://img-cdn.pixlr.com/image-generator/history/65bb506dcb310754719cf81f/ede935de-1138-4f66-8ed7-44bd16efc709/medium.webp',
  },
  {
    title: 'Product 2',
    price: 200,
    rating: 4,
    imageUrl:
      'https://img-cdn.pixlr.com/image-generator/history/another-product/medium.webp',
  },
  {
    title: 'Product 3',
    price: 300,
    rating: 4.5,
    imageUrl:
      'https://img-cdn.pixlr.com/image-generator/history/third-product/medium.webp',
  },
];

export const MOCK_PRODUCTS_TOP_SELLING = [
  {
    id: 1,
    attributes: {
      price: 1000,
      createdAt: '2024-08-27T04:20:56.562Z',
      updatedAt: '2024-09-23T02:52:07.552Z',
      publishedAt: '2024-09-10T04:14:24.732Z',
      rating: 4,
      title: 'I phone 12',
      imageUrl:
        'https://images.unsplash.com/photo-1505740420928-5e560c06d30e?fm=jpg&q=60&w=3000&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8Nnx8cHJvZHVjdHxlbnwwfHwwfHx8MA%3D%3D',
      brand: 'Apple',
      negotiable: true,
      description:
        'This the New creation Of apple  This the New creation Of apple This the New creation Of apple This the New creation Of apple.',
    },
  },
  {
    id: 2,
    attributes: {
      price: 1200,
      createdAt: '2024-08-27T04:21:20.293Z',
      updatedAt: '2024-09-23T02:58:11.083Z',
      publishedAt: '2024-09-10T04:14:40.546Z',
      rating: 4,
      title: 'Macbook m1',
      imageUrl:
        'https://images.rawpixel.com/image_800/cHJpdmF0ZS9sci9pbWFnZXMvd2Vic2l0ZS8yMDIyLTExL3JtMzYyLTAxYS1tb2NrdXAuanBn.jpg',
      brand: 'Apple',
      negotiable: false,
      description:
        'This the New creation Of apple  This the New creation Of apple This the New creation Of apple This the New creation Of apple.',
    },
  },
  {
    id: 3,
    attributes: {
      price: 1000,
      createdAt: '2024-09-09T07:45:09.821Z',
      updatedAt: '2024-09-23T02:53:33.888Z',
      publishedAt: '2024-09-10T04:14:34.734Z',
      rating: 2,
      title: 'Samsung A13',
      imageUrl:
        'https://img.freepik.com/free-photo/organic-cosmetic-product-with-dreamy-aesthetic-fresh-background_23-2151382816.jpg',
      brand: 'Samsung',
      negotiable: true,
      description:
        "Discover Samsung's newest creation, combining innovation with cutting-edge technology. Experience the future with enhanced performance and sleek design.",
    },
  },
  {
    id: 4,
    attributes: {
      price: 1500,
      createdAt: '2024-09-09T07:45:45.847Z',
      updatedAt: '2024-09-23T02:54:18.198Z',
      publishedAt: '2024-09-10T04:14:46.763Z',
      rating: 4.6,
      title: 'Macbook pro m3',
      imageUrl:
        'https://images.unsplash.com/photo-1523275335684-37898b6baf30?fm=jpg&q=60&w=3000&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8Mnx8cHJvZHVjdHxlbnwwfHwwfHx8MA%3D%3D',
      brand: 'Apple',
      negotiable: false,
      description:
        'This the New creation Of apple  This the New creation Of apple This the New creation Of apple This the New creation Of apple.',
    },
  },
  {
    id: 5,
    attributes: {
      price: 3,
      createdAt: '2024-09-10T04:18:27.070Z',
      updatedAt: '2024-09-23T02:57:58.388Z',
      publishedAt: '2024-09-10T04:18:27.725Z',
      rating: 2,
      title: 'Samsung Laptop',
      imageUrl:
        'https://images.unsplash.com/photo-1505740420928-5e560c06d30e?fm=jpg&q=60&w=3000&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8Nnx8cHJvZHVjdHxlbnwwfHwwfHx8MA%3D%3D',
      brand: 'Samsung',
      negotiable: false,
      description:
        "Discover Samsung's newest creation, combining innovation with cutting-edge technology. Experience the future with enhanced performance and sleek design.",
    },
  },
];

export const MOCK_INVOICE_PRODUCTS = [
  {
    id: 5,
    attributes: {
      price: 1600,
      createdAt: '2024-09-22T22:21:05.920Z',
      updatedAt: '2024-09-23T02:58:57.822Z',
      publishedAt: '2024-09-22T22:21:05.914Z',
      quantity: 1,
    },
  },
  {
    id: 6,
    attributes: {
      price: 1000,
      createdAt: '2024-09-22T22:22:39.895Z',
      updatedAt: '2024-09-22T22:22:39.895Z',
      publishedAt: '2024-09-22T22:22:39.893Z',
      quantity: 1,
    },
  },
  {
    id: 7,
    attributes: {
      price: 1600,
      createdAt: '2024-09-22T22:23:25.215Z',
      updatedAt: '2024-09-23T02:59:05.936Z',
      publishedAt: '2024-09-22T22:23:25.212Z',
      quantity: 1,
    },
  },
  {
    id: 8,
    attributes: {
      price: 1400,
      createdAt: '2024-09-22T22:24:22.140Z',
      updatedAt: '2024-09-23T02:59:14.013Z',
      publishedAt: '2024-09-22T22:24:22.139Z',
      quantity: 2,
    },
  },
  {
    id: 9,
    attributes: {
      price: 1000,
      createdAt: '2024-09-22T22:25:15.043Z',
      updatedAt: '2024-09-22T22:25:15.043Z',
      publishedAt: '2024-09-22T22:25:15.041Z',
      quantity: 1,
    },
  },
];

export const MOCK_PRODUCTS: (IProduct & { id: number })[] = [
  {
    id: 1,
    imageUrl:
      'https://images.unsplash.com/photo-1505740420928-5e560c06d30e?fm=jpg&q=60&w=3000&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8Nnx8cHJvZHVjdHxlbnwwfHwwfHx8MA%3D%3D',
    title: 'Iphone 12',
    price: 1000,
    rating: 4,
    createdAt: '2024-08-27T04:20:56.562Z',
    updatedAt: '2024-09-23T02:52:07.552Z',
    publishedAt: '2024-09-10T04:14:24.732Z',
    totalSale: 100,
  },
  {
    id: 2,
    imageUrl:
      'https://images.unsplash.com/photo-1505740420928-5e560c06d30e?fm=jpg&q=60&w=3000&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8Nnx8cHJvZHVjdHxlbnwwfHwwfHx8MA%3D%3D',
    title: 'Iphone 13',
    price: 2000,
    rating: 5,
    createdAt: '2024-09-22T22:25:15.043Z',
    updatedAt: '2024-09-22T22:25:15.043Z',
    publishedAt: '2024-09-22T22:25:15.041Z',
    totalSale: 969,
  },
];
