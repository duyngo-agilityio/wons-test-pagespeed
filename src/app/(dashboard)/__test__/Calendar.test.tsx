// Mocks
import { EVENT_TABS } from '@/mocks';

// Page
import CalendarPage from '../calendar/page';

jest.mock('@/api', () => ({
  ...jest.requireActual('@/api'),
  getCalendarEvents: jest.fn(() => ({
    data: EVENT_TABS,
  })),
}));

describe('Calendar page', () => {
  it.skip('should match snapshot', async () => {
    const { container } = testLibJestUtils.render(<CalendarPage />);

    await testLibJestUtils.waitFor(() => {
      expect(container).toMatchSnapshot();
    });
  });
});
