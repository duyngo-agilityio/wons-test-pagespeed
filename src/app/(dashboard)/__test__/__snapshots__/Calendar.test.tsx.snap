// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar page should match snapshot 1`] = `
<div>
  <div
    data-overlay-container="true"
  >
    <script>
      !function(){try{var d=document.documentElement,c=d.classList;c.remove('light','dark');var e=localStorage.getItem('theme');if('system'===e||(!e&&false)){var t='(prefers-color-scheme: dark)',m=window.matchMedia(t);if(m.media!==t||m.matches){d.style.colorScheme = 'dark';c.add('dark')}else{d.style.colorScheme = 'light';c.add('light')}}else if(e){c.add(e|| '')}else{c.add('light')}if(e==='light'||e==='dark'||!e)d.style.colorScheme=e||'light'}catch(e){}}()
    </script>
    <section>
      <div
        class="base:mb-6 md:mb-7.5 w-full md:flex justify-between md:items-center"
      >
        <h2
          aria-level="2"
          class="font-bold font-dm-sans text-blue-800 dark:text-white text-5xl w-full"
        >
          Calendar
        </h2>
      </div>
    </section>
  </div>
</div>
`;
